content/id/examples/service/networking/dual-stack-preferred-svc.yaml

```diff
@@ -3,10 +3,10 @@ kind: Service
 metadata:
   name: my-service
 spec:
-  ipFamily: IPv6
+  ipFamilyPolicy: PreferDualStack
   selector:
     app: MyApp
   ports:
     - protocol: TCP
       port: 80
-      targetPort: 9376
\ No newline at end of file
+      targetPort: 9376
```

Comment:
So do we still want this example port? It adds possible ambiguity.

Reply:
Yes we should remove any fields that aren't necessary.